src/ui/public/kbn_top_nav/bread_crumbs/bread_crumb_urls.js

```diff
@@ -14,7 +14,7 @@
  */
 export function getBreadCrumbUrls(breadcrumbs, url) {
   return breadcrumbs.map(breadcrumb => {
-    const breadCrumbStartIndex = url.toLowerCase().lastIndexOf(breadcrumb.toLowerCase());
+    const breadCrumbStartIndex = url.toLowerCase().lastIndexOf(breadcrumb.replace(/\W/, '').toLowerCase());
```

Comment:
> `breadcrumb.replace(/\W/, '').toLowerCase()`

If I'm understanding this code snippet right, it is taking `breadcrumb` (which is a string), replacing the first non-word character with a `''`, and then converting it to lower case. Why is this necessary?